src/client/websocket/packets/handlers/UserGuildSettingsUpdate.js

```diff
@@ -13,7 +13,7 @@ class UserGuildSettingsUpdateHandler extends AbstractHandler {
 }
 
 /**
- * Emitted whenever the client user's settings update.
+ * Emitted whenever the client user's guild update.
```

Comment:
As the name says, this event is fired when the client user's settings for a guild updates, this description can be confused with [guildUpdate](https://discord.js.org/#/docs/main/stable/class/Client?scrollTo=e-guildUpdate)

Reply:
maybe, but this can also be confused with [clientUserSettingsUpdate](https://discord.js.org/#/docs/main/stable/class/Client?scrollTo=e-clientUserSettingsUpdate)